app/controllers/reservations_controller.rb

```diff
@@ -94,7 +94,8 @@ def list
   # POST /orders/1/order_details/1/reservations
   def create
     raise ActiveRecord::RecordNotFound unless @reservation.nil?
-    @reservation = @order_detail.build_reservation(reservation_create_params)
+    @reservation = @order_detail.build_reservation
+    @reservation.assign_attributes(reservation_create_params)
```

Comment:
This was already a sanitization issue before. We can deal with this when we do the rails upgrade.